hack/boilerplate/boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (C) YEAR Red Hat, Inc.
+COPYRIGHTHOLDER.
```

Comment:
I was first going to ask why you ended up to replace the whole line with this token and not just the copy right holder itself. But I get it now. Looking at _boilerplate.py_, I have to say I find this a strange way of doing in general. But let's be done with it for now.